ros2_batch_job/__main__.py

```diff
@@ -48,7 +48,8 @@
     'pep8',
     'pydocstyle',
     'pyflakes',
-    'flake8',
+    'flake8==2.6.2',
```

Comment:
What is the reason to pin the version to 2.x instead of using the latest release - currently 3.0.4?